std/array.d

```diff
@@ -4011,6 +4031,43 @@ if (isDynamicArray!A)
     app2.toString();
 }
 
+// https://issues.dlang.org/show_bug.cgi?id=24856
+@system unittest
+{
+    import core.memory : GC;
+    import std.stdio : writeln;
+    import std.algorithm : canFind;
```

Comment:
```
Check for package wide std.algorithm imports
grep -nr 'import std.algorithm : ' $(find etc std -name '*.d') ; test $? -eq 1
std/array.d:4039:    import std.algorithm : canFind;
```
I think it wants `import std.algorithm.searching : canFind;`

Reply:
oh how quaint...